### DISCUSSION TITLE
PIPE Error message

### DISCUSSION THREAD

**Discussion by Remski** [abraunegg/onedrive#1677] [Q&A]
Hello all,

Have finally gotten round to installing this on Manjaro gnome

Got everything working well and managed to sync a few directories (manually for now)

However I did quite a bit of tidying up once replicated locally (using DupeGuru to remove dupes and then re-arranging files where the should go

Nopw I keep getting what I suspect is a timeout whenever I try to sync back the changes

The error message is simply PIPE x" with a timestamp

![image](https://user-images.githubusercontent.com/76783886/140663004-70742286-cc50-41e4-a21c-9ed258e3afd3.png)

Can any one advise on what this means and what I can do to prevent it in future?

Many thanks
R

**Comment by abraunegg** [comment]
@Remski 
You need increase your logging verbosity to see what is going on, additionally, some critical points are missing from your information such as what client version you are using, how are you running the application in Manjaro and so on.

Please run the application in 'verbose' mode by adding `--verbose` to your command line to see if there is anything more informative being provided. If this is not providing anything informative, please generate a debug log as per https://github.com/abraunegg/onedrive/wiki/Generate-debug-log-for-support

The application does not use any PIPE or anything like that - so if you are getting this error as an actual error message it is really odd, and potentially not coming from this application at all.
